R/bundle.R

```diff
@@ -449,28 +456,77 @@ inferDependencies <- function(appMode, hasParameters) {
   if (appMode == 'api') {
     deps <- c(deps, "plumber")
   }
+  if (!is.null(python)) {
+    deps <- c(deps, "reticulate")
+  }
   unique(deps)
 }
 
+inferPythonEnv <- function(workdir, python) {
+  # run the python introspection script
+  env_py <- system.file("resources/environment.py", package = "rsconnect")
+  args <- c(env_py, workdir)
```

Comment:
We might need to use `shQuote()` here to ensure arguments are shell-quoted as appropriate. (Unfortunately the `system2()` interface doesn't quote arguments passed in)